lib/models/login/login_response_model.dart

```diff
@@ -0,0 +1,35 @@
+class LoginResponseModel {
+  bool status;
+  String message;
+  UserData data;
+
+  LoginResponseModel.fromJson(Map<String, dynamic> json) {
+    status = json['status'] as bool;
+    message = json['message'] as String;
+    data = json['data'] != null
+        ? UserData.fromJson(json['data'])
+        : null;
+  }
+}
+
+class UserData {
```

Comment:
separate User Data class in anther file and try to use better name

Reply:
done